kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/io/kotest/engine/DispatchersSharedCoroutineSchedulerTest.kt

```diff
@@ -0,0 +1,26 @@
+package io.kotest.engine
+
+import io.kotest.core.spec.IsolationMode
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.core.test.testCoroutineScheduler
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.delay
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class DispatchersSharedCoroutineSchedulerTest : WordSpec({
+   coroutineTestScope = true
+   isolationMode = IsolationMode.InstancePerLeaf
+
+   "a" should {
```

Comment:
should we replace `"a"` and `"b"`  with  more descriptive test names?

Reply:
sure